src/sql/workbench/contrib/notebook/browser/markdownWYSIWYGLinkHander.ts

```diff
@@ -0,0 +1,91 @@
+import { URI } from 'vs/base/common/uri';
+import * as path from 'vs/base/common/path';
+
+export class MarkdownWYSIWYGLinkHandler {
+	private _isAnchorLink: boolean | undefined;
+	private _isFile: boolean | undefined;
+	private _isAbsolutePath: boolean | undefined;
+	private _notebookDirectory: string;
+	private _href: URI | undefined;
+
+	constructor(private _notebookURI: URI, private _path: string | HTMLAnchorElement) {
```

Comment:
path doesn't really seem correct - maybe just `pathOrElement` just to be clearer?